language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/package.scala

```diff
@@ -23,7 +23,8 @@ package object rxjava {
   private[rxjava] def untestedEndpoint: Nothing =
     throw new UnsupportedOperationException("Untested endpoint, implement if needed")
 
-  private[rxjava] val authorizer = new Authorizer(() => Clock.systemUTC().instant())
```

Comment:
Not a problem with this PR, but is there a reason the `Authorizer` takes a `() => Instant` rather than a `Clock`?

Reply:
There is (was?) one case in which we needed to pass a different type of object and this ensured we didn't have to convert it to a `Clock` to make this work.